From 4c0d15f86f33dc61ec1dc618fb164c127409edfc Mon Sep 17 00:00:00 2001
From: Min-Yih Hsu <minyihh@uci.edu>
Date: Sat, 7 Aug 2021 17:07:28 -0700
Subject: [PATCH] Update `llvm-readobj` command invocation in
 extract-section.py

Change `-elf-output-style` to `--elf-output-style` to reflect the recent
changes on short/long CLI options in LLVM binary utils.
---
 llvm/utils/extract-section.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/llvm/utils/extract-section.py b/llvm/utils/extract-section.py
index ca19b0ee8b61..0c96b52fc522 100755
--- a/llvm/utils/extract-section.py
+++ b/llvm/utils/extract-section.py
@@ -31,7 +31,7 @@ def read_raw_stdin():
 
 def get_raw_section_dump(readobj_path, section_name, input_file):
     import subprocess
-    cmd = [readobj_path, '-elf-output-style=GNU', '--hex-dump={}'.format(section_name),
+    cmd = [readobj_path, '--elf-output-style=GNU', '--hex-dump={}'.format(section_name),
             input_file]
     proc = subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
 
-- 
2.26.2

